modules/imgproc/src/approx.cpp

```diff
@@ -103,9 +103,9 @@ CvSeq* icvApproximateChainTC89( CvChain* chain, int header_size,
         /* calc 1-curvature */
         s = abs_diff[reader.code - prev_code + 7];
 
-        if( method <= cv::CHAIN_APPROX_SIMPLE )
+        if( method <= CV_CHAIN_APPROX_SIMPLE )
```

Comment:
Why do you revert C++ const to old C const?

Reply:
Reverted to older version by mistake, I'll fix it.